nginx.conf

```diff
@@ -13,13 +13,104 @@
+      application/mac-binhex40              hqx;
+      application/msword                    doc;
+      application/pdf                       pdf;
+      application/postscript                ps eps ai;
+      application/rtf                       rtf;
+      application/vnd.apple.mpegurl         m3u8;
+      application/vnd.ms-excel              xls;
+      application/vnd.ms-fontobject         eot;
+      application/vnd.ms-powerpoint         ppt;
+      application/vnd.wap.wmlc              wmlc;
+      application/vnd.google-earth.kml+xml  kml;
+      application/vnd.google-earth.kmz      kmz;
+      application/x-7z-compressed           7z;
+      application/x-cocoa                   cco;
+      application/x-java-archive-diff       jardiff;
+      application/x-java-jnlp-file          jnlp;
+      application/x-makeself                run;
+      application/x-perl                    pl pm;
+      application/x-pilot                   prc pdb;
+      application/x-rar-compressed          rar;
+      application/x-redhat-package-manager  rpm;
+      application/x-sea                     sea;
+      application/x-shockwave-flash         swf;
+      application/x-stuffit                 sit;
+      application/x-tcl                     tcl tk;
+      application/x-x509-ca-cert            der pem crt;
+      application/x-xpinstall               xpi;
+      application/xhtml+xml                 xhtml;
+      application/xspf+xml                  xspf;
+      application/zip                       zip;
+
+      application/octet-stream              bin exe dll;
+      application/octet-stream              deb;
+      application/octet-stream              dmg;
+      application/octet-stream              iso img;
+      application/octet-stream              msi msp msm;
+
+      application/vnd.openxmlformats-officedocument.wordprocessingml.document    docx;
+      application/vnd.openxmlformats-officedocument.spreadsheetml.sheet          xlsx;
+      application/vnd.openxmlformats-officedocument.presentationml.presentation  pptx;
+
+      audio/midi                            mid midi kar;
+      audio/mpeg                            mp3;
+      audio/ogg                             ogg;
+      audio/x-m4a                           m4a;
+      audio/x-realaudio                     ra;
+
+      video/3gpp                            3gpp 3gp;
+      video/mp2t                            ts;
+      video/mp4                             mp4;
+      video/mpeg                            mpeg mpg;
+      video/quicktime                       mov;
+      video/webm                            webm;
+      video/x-flv                           flv;
+      video/x-m4v                           m4v;
+      video/x-mng                           mng;
+      video/x-ms-asf                        asx asf;
+      video/x-ms-wmv                        wmv;
+      video/x-msvideo                       avi;
+  }
```

Comment:
I doubt this is all necessary - it's just a copy of my local `mime.types`

Reply:
Sounds good. :+1: I like the idea of specifying these inline rather than including a file whose path changes based on the OS.